libs/application/templates/university/src/forms/UniversityForm/FormerEducation/OtherDocuments.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  buildFileUploadField,
+  buildMultiField,
+  buildSubSection,
+} from '@island.is/application/core'
+import { formerEducation } from '../../../lib/messages/formerEducation'
+import { Routes } from '../../../lib/constants'
+import { FILE_SIZE_LIMIT } from '../../../shared'
+import { FormValue } from '@island.is/application/types'
+import { UniversityApplication } from '../../../lib/dataSchema'
+
+export const OtherDocumentsSubSection = buildSubSection({
+  id: Routes.OTHERDOCUMENTS,
+  title: formerEducation.general.sectionTitle,
+  children: [
+    buildMultiField({
+      id: 'otherDocumentsMultiField',
+      title: '',
+      children: [
+        buildFileUploadField({
+          id: `${Routes.OTHERDOCUMENTS}.degreeAttachments`,
+          title: formerEducation.labels.educationDetails.degreeFileUploadTitle,
+          introduction: '',
+          maxSize: FILE_SIZE_LIMIT,
+          condition: (formValue: FormValue, externalData) => {
+            const answers = formValue as UniversityApplication
+            const chosenOption = answers.educationOptions
+            return true
```

Comment:
Because we are waiting for fields from the universities that tell us if they require other documents, but I can just delete this file for now if that is better